### PR TITLE
Modify the code format (video mode related)

### PR DESCRIPTION
**Pull request by Dragon-S**

### CODE REVIEW

File: src/MovieMator/src/dialogs/videomodesettingsdialog.cpp [hunk 1]
```diff
@@ -9,7 +9,8 @@
 #include <qstandardpaths.h>
 #include <qmessagebox.h>
 
-const QMap<QString, QString> profileNameMap = {
+const QMap<QString, QString> profileNameMap =
+{
     {"", "Automatic"},
```
Comment (vgawen) [inline src/MovieMator/src/dialogs/videomodesettingsdialog.cpp]: 对齐

File: src/MovieMator/src/dialogs/videomodesettingsdialog.cpp [hunk 2]
```diff
@@ -66,32 +67,41 @@ VideoModeSettingsDialog::~VideoModeSettingsDialog()
...
-    QString currentPrefileFileName = Settings.playerProfile();
-    QList<QString> keys =  profileNameMap.keys();
-    if (keys.contains(currentPrefileFileName)) {
-        ui->videoModeComboBox->setCurrentText(profileNameMap.value(currentPrefileFileName));
-    } else {
-        ui->videoModeComboBox->setCurrentText(currentPrefileFileName);
+void VideoModeSettingsDialog::setupCurrentProfileUI()
+{
+    QString strCurrentPrefileFileName   = Settings.playerProfile();
+    QList<QString> keys                 =  profileNameMap.keys();
```
Comment (vgawen) [inline src/MovieMator/src/dialogs/videomodesettingsdialog.cpp]: =右边多了空格

Comment (Dragon-S) [reply src/MovieMator/src/dialogs/videomodesettingsdialog.cpp]:
```suggestion
    QList<QString> keys                 = profileNameMap.keys();
```

File: src/MovieMator/src/dialogs/videomodesettingsdialog.cpp (hunk 1 above)
Comment (Dragon-S) [reply src/MovieMator/src/dialogs/videomodesettingsdialog.cpp]:
```suggestion
`    {"",                    "Automatic"},
    {"atsc_720p_50",        "HD 720p 50 fps"},
    {"atsc_720p_5994",      "HD 720p 59.94 fps"},
    {"atsc_720p_60",        "HD 720p 60 fps"},
    {"atsc_1080i_50",       "HD 1080i 25 fps"},
    {"atsc_1080i_5994",     "HD 1080i 29.97 fps"},
    {"atsc_1080p_2398",     "HD 1080p 23.98 fps"},
    {"atsc_1080p_24",       "HD 1080p 24 fps"},
    {"atsc_1080p_25",       "HD 1080p 25 fps"},
    {"atsc_1080p_2997",     "HD 1080p 29.97 fps"},
    {"atsc_1080p_30",       "HD 1080p 30 fps"},
    {"dv_ntsc",             "SD NTSC"},
    {"dv_pal",              "SD PAL"},
    {"uhd_2160p_2398",      "UHD 2160p 23.98 fps"},
    {"uhd_2160p_24",        "UHD 2160p 24 fps"},
    {"uhd_2160p_25",        "UHD 2160p 25 fps"},
    {"uhd_2160p_2997",      "UHD 2160p 29.97 fps"},
    {"uhd_2160p_30",        "UHD 2160p 30 fps"},
    {"uhd_2160p_50",        "UHD 2160p 50 fps"},
    {"uhd_2160p_5994",      "UHD 2160p 59.94 fps"},
    {"uhd_2160p_60",        "UHD 2160p 60 fps"},`
```

**Pull request merged**